apps/web/components/ui/PurchaseCard/PurchaseCard.vue

```diff
@@ -27,11 +27,18 @@
         />
       </div>
     </div>
-    <Price
-      :price="currentActualPrice"
-      :normal-price="normalPrice"
-      :old-price="productGetters.getPrice(product).regular ?? 0"
-    />
+    <div class="flex space-x-2">
+      <Price
+        :price="currentActualPrice"
+        :normal-price="normalPrice"
+        :old-price="productGetters.getPrice(product).regular ?? 0"
+      />
+      <div v-if="(productBundleGetters?.getBundleDiscount(product) ?? 0) > 0" class="m-auto">
```

Comment:
should make separate getter like existingBundleDiscount, but can be done in next iteration.